student_explorer/settings.py

```diff
@@ -190,10 +190,11 @@
     SAML2_REMOTE_PEM_FILE = config('STUDENT_EXPLORER_SAML_REMOTE_PEM_FILE',default='')
 
     SAML2_DEFAULT_IDP = config('STUDENT_EXPLORER_SAML_DEFAULT_IDP',default='')
-
     if SAML2_DEFAULT_IDP:
         SAML2_DEFAULT_IDP = '?idp=%s' % SAML2_DEFAULT_IDP
 
+    SAML2_ENTITYID = config('SAML2_ENTITYID', default=SAML2_URL_BASE + 'metadata/')
```

Comment:
Looks like we don't mention any of these other SAML values in the sample. We probably should document them somewhere but that could be on another ticket.

Reply:
Yeah, I was wondering about that. I thought maybe it had something to do with how we configure environment variables through OpenShift (i.e. how we aren't set up to use a Config Map). I think we can create another issue that addresses this (and maybe what I tried to sneak in but @zqian called me out on).